src/main/resources/testdata/import-testdata.sh

```diff
@@ -36,30 +41,49 @@ DB_HOST="127.0.0.1"
 DB_PORT="3306"
 
 # Path to testdata SQL files
-SQL_PATH="./"
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+SQL_PATH="$SCRIPT_DIR"
 
-echo "🚀 Importing SQL test data into MySQL database '$DB_NAME'..."
+echo "Importing SQL test data into MySQL database '$DB_NAME'..."
+echo "Searching for SQL files in: $SQL_PATH"
 
 # Check for mysql CLI
 if ! command -v mysql &> /dev/null
 then
-  echo "❌ mysql CLI not found. Please install MySQL client."
+  echo "mysql CLI not found. Please install MySQL client."
   exit 1
 fi
 
+# Ask user if they want to reset the DB (run drop script)
+echo "Do you want to reset the database (run 00_drop_all_tables.sql)? (y/n)"
+read -r RESET_DB
+if [[ "$RESET_DB" == "y" || "$RESET_DB" == "Y" ]]; then
+  DROP_FILE="$SQL_PATH/00_drop_all_tables.sql"
+  if [ -f "$DROP_FILE" ]; then
+    echo "Resetting database..."
+    mysql -h "$DB_HOST" -P "$DB_PORT" -u "$DB_USER" --password="$DB_PASS" "$DB_NAME" < "$DROP_FILE"
+  else
+    echo "Reset script not found at: $DROP_FILE"
+    exit 1
+  fi
+else
+  echo "Skipping database reset."
+fi
```

Comment:
```suggestion
while true; do
  echo "Do you want to reset the database (run 00_drop_all_tables.sql)? (y/n)"
  read -p "Do you want to reset the database? (y/n): " RESET_DB
  if [[ "$RESET_DB" == "y" || "$RESET_DB" == "Y" ]]; then
    DROP_FILE="$SQL_PATH/00_drop_all_tables.sql"
    if [ -f "$DROP_FILE" ]; then
      echo "Resetting database..."
      mysql -h "$DB_HOST" -P "$DB_PORT" -u "$DB_USER" --password="$DB_PASS" "$DB_NAME" < "$DROP_FILE"
    else
      echo "Reset script not found at: $DROP_FILE"
      exit 1
    fi
    break
  elif [[ "$RESET_DB" == "n" || "$RESET_DB" == "N" ]]; then
    echo "Skipping database reset."
    break
  else
    echo "Invalid input. Please enter y or n."
  fi
done
```

Suggestion: at the moment, if someone enters anything else except 'y' or 'Y', it is interpreted as 'n'/'N'.
Standard behaviour would dictate to ask again.

Reply:
Thank you, fixed in the latest commit 👍🏼